### PR TITLE
VB-3941 Update booking journey URLs and refactor code to match

### PR DESCRIPTION
**Pull request by tpmcgowan**

### CODE REVIEW

File: server/routes/bookVisit/additionalSupportController.ts [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 
   public view(): RequestHandler {
     return async (req, res) => {
-      res.render('pages/bookingJourney/selectAdditionalSupport', {
+      res.render('pages/bookVisit/additionalSupport', {
         errors: req.flash('errors'),
         formValues: req.flash('formValues')?.[0] || {},
-        booker: req.session.booker,
-        bookingJourney: req.session.bookingJourney,
+        prisonName: req.session.bookingJourney.prison.prisonName,
```
Comment (tpmcgowan) [inline server/routes/bookVisit/additionalSupportController.ts:12]: Unrelated but spotted we were passing unnecessary data to template: only prison name actually used.

File: server/routes/bookVisit/index.ts
Comment (hutcheonb-moj) [inline server/routes/bookVisit/index.ts:1]: Only comment would be should we not change the variable names of the controllers to match their new actual name? selectVisitDateTimeController => chooseVisitTimeController for instance

**Review by hutcheonb-moj — DISMISSED** [review]
LGTM - One suggestion made

File: server/routes/bookVisit/index.ts
Comment (tpmcgowan) [reply server/routes/bookVisit/index.ts:1]: Good spot - fixed!

**Review by hutcheonb-moj — APPROVED** [review]
LGTM

**Pull request merged**